### PR TITLE
Apply package authoring best practices for NuGet

### PR DESCRIPTION
**Pull request by Corniel**
To follow the best practices as specified by Microsoft:
- [ ] Consider choosing a NuGet package name with a prefix that meets NuGet's prefix reservation criteria.
- [x] Consider using SemVer to version your NuGet package.
- [x] Do use the author field for your or your organization's "pretty name."
- [x] Do include a short description (up to 4000 characters) to describe your package.
- [x] Do add a copyright notice to your package with "Copyright (c) <name/company> <year>."
- [x] DO include a link to an associated project, repository, or company website.
- [x] Consider including an icon with your package to help visually differentiate it. It's a relatively small addition that can improve perception of package quality.
- [x] Do use an image that is 128x128 and has a transparent background (PNG) for best viewing results.
- [x] Do add a README markdown file that provides an overview of what your package does and how to get started.
- [x] Consider setting up Source Link to automatically add source control metadata to your NuGet package and make your library easier to debug.
- [x] Do include several tags with key terms related to your package to enhance discoverability.
- [x] Do include release notes with each update describing what changes were made.
- [x] Do include a license expression or license file in your package.
- [x] Do choose an open source license to make your package open source.
- [x] Consider including a license expression in your package.
- [x] Do include a license file in your package.

https://learn.microsoft.com/en-us/nuget/create-packages/package-authoring-best-practices

### CODE REVIEW

File: props/common.props [hunk 1]
```diff
@@ -2,10 +2,32 @@
 <Project>
 
   <PropertyGroup>
-    <Description>Qowaiv implements common, universal domain objects. These types form the base of your domain model.</Description>
+    <Title>Qowaiv (Single) Value Object library</Title>
+    <Description>
+      Qowaiv is a (Single) Value Object library. It aims to model reusable (Single)
+      Value Objects that can be used a wide variety of modeling scenarios, both
```
Comment (LauraXiulan) [inline props/common.props:8]: **in** a wide variety

File: props/common.props [hunk 2]
```diff
@@ -2,10 +2,32 @@
...
+      inside and outside a Domain-driven context.
+    </Description>
     <Authors>Ad van der Hoeven;Corniel Nobel;Erik Ammerlaan;Fons Sonnemans;Jack Kester;Hanif Hafez Nezami;Laura Kramer;Patrick Evers;Vincent Lesierse;Wilko Frieke</Authors>
     <Owners>Qowaiv community</Owners>
-    <PackageTags>qowaiv domain model</PackageTags>
+    <PackageTags>
```
Comment (LauraXiulan) [inline props/common.props:13]: "yes no", "country", "identifiers", "security", "serialization", "fluent", "clock"
You can also add tags related to the type of SVO's you have, so: chemistry, financial, globalization, mathematical etc. Or is that not specific enough?

File: src/Qowaiv.Data.SqlClient/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿# Qowaiv.Data.SqlClient
+
+## Domain-driven design bottom up
+Qowaiv is a (Single) Value Object library. It aims to model reusable, immutable,
+(Single) Value Objects that can be used a wide variety of modeling scenarios,
```
Comment (LauraXiulan) [inline src/Qowaiv.Data.SqlClient/README.md:5]: Same change as previous comment, also for the Readme's in TestTools and Qowaiv

**Review by LauraXiulan — APPROVED** [review]

File: props/common.props (hunk 2 above)
Comment (Corniel) [reply props/common.props:13]: I'm not sure. I like the idea, but should we hope/expect that people searching for those terms find this package?

Comment (LauraXiulan) [reply props/common.props:13]: Hmm, more tags is more visibility, but if they are too generic, Qowaiv will not stand out from the many results.

**Pull request merged**